<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CifCreatorFrame</class>
 <widget class="QWidget" name="CifCreatorFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0">
   <item row="6" column="1">
    <widget class="tabPanel" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,0,0,0,0">
       <item row="6" column="2">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Horizontal axis: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>γ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="11">
        <layout class="QVBoxLayout" name="vLayoutLeft" stretch="1,0">
         <item>
          <layout class="QVBoxLayout" name="vPlotLayout"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
           <item>
            <widget class="QComboBox" name="cmbViewDirection">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Top</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Front</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Right</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Back</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Left</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnPreview">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Preview</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="9" column="5">
        <widget class="EditUnitsBox" name="edtTiltC">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="EditUnitsBox" name="edtZoneB">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Dimensions: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <spacer name="hSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="4">
        <widget class="EditUnitsBox" name="edtZoneV">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="4">
        <widget class="Line" name="line_2">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="EditUnitsBox" name="edtRangeX">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tilt: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Zone axis: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="EditUnitsBox" name="edtTiltA">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="hSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>u</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>v</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="EditUnitsBox" name="edtZoneC">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="EditUnitsBox" name="edtRangeZ">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="EditUnitsBox" name="edtZoneU">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="4">
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="EditUnitsBox" name="edtTiltB">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="EditUnitsBox" name="edtRangeY">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>z</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="5">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>w</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>β</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="EditUnitsBox" name="edtZoneA">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>α</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="EditUnitsBox" name="edtZoneW">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditUnitsBox</class>
   <extends>QLineEdit</extends>
   <header>controls/editunitsbox.h</header>
  </customwidget>
  <customwidget>
   <class>tabPanel</class>
   <extends>QTabWidget</extends>
   <header>controls/tabpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cmbViewDirection</tabstop>
  <tabstop>btnPreview</tabstop>
  <tabstop>edtRangeX</tabstop>
  <tabstop>edtRangeY</tabstop>
  <tabstop>edtRangeZ</tabstop>
  <tabstop>edtZoneU</tabstop>
  <tabstop>edtZoneV</tabstop>
  <tabstop>edtZoneW</tabstop>
  <tabstop>edtZoneA</tabstop>
  <tabstop>edtZoneB</tabstop>
  <tabstop>edtZoneC</tabstop>
  <tabstop>edtTiltA</tabstop>
  <tabstop>edtTiltB</tabstop>
  <tabstop>edtTiltC</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
